deps/wazuh_testing/wazuh_testing/data_visualization_module/README.md

```diff
@@ -0,0 +1,141 @@
+# Data Visualization Module
+
+This module contains a basic application developed with Dash and Plotly that allows to visualize different data from CSV files in a simple, representative and easily interpretable way. It allows you to dynamically choose which type of data to display and to which file it belongs, making data comparison much easier.
+
+## Initial setup
+
+To run these tests, we need to install the package. So we can follow these steps:
+
+1. Move to the `wazuh_testing/data_visualization_module` directory.
+2. Create and activate the Python environment.
+
+```shell script
+python3 -m venv env
+source env/bin/activate
+```
+
+3. Install the package
+
+```shell script
+python3 -m pip install .
+```
+
+## Run module
+
+To run this module, first, we need to add the data we want to display to the database. For more information about this, you should read the Database section, and run the following command:
+
+```shell script
+python3 scripts/db_data_insertion.py <directory> <component>
+```
+
+Then, we will need to use the following command to launch the app:
+
+```shell script
+python3 app.py --config items_config.yml
+```
+
+When the module is running, we must access `http://127.0.0.1:8050/` in a browser to view the data.
+
+### Parameters
+
+- `--config`: YAML file containing the component and the processes to visualize. It also includes those CSV columns that are not required for visualization and should be omitted.
+
+An example of the structure of this file could be the following:
+
+```shell script
+Component:
+  - "manager"
+
+Processes:
+  Daemon:
+    - "wazuh-clusterd"
+    - "wazuh-integratord"
+    - "wazuh-execd"
+    - "wazuh-logcollector"
+    - "wazuh-analysisd"
+    - "wazuh-db"
+    - "wazuh-maild"
+    - "wazuh-authd"
+    - "wazuh-syscheckd"
+    - "wazuh-monitord"
+    - "wazuh-modulesd"
+    - "wazuh-remoted"
+
+Columns_to_avoid:
+  - "Daemon"
+  - "Version"
+  - "Timestamp"
+  - "Commit"
+```
+
+The `Component` value can take one of these: [`manager`, `indexer`, `agent` or `dashboard`].
```

Comment:
The component should be more flexible. Data could come from managers of different OS, resources, or tests cases. We need to improve database structure and standardized data schema.

Reply:
We will handle it in a data standardization and database improvement issue